### PR TITLE
Setup 2fa from login

### PR DESCRIPTION
**Pull request by pandey-shubh**
* added test to setup 2fa from login flow

### CODE REVIEW

**Comment by pandey-shubh** [comment]
[
Screenshot 2022-06-30 at 12 44 54 PM
](url)

File: openlogin/setup-2fa/index.config.ts [hunk 1]
```diff
@@ -11,15 +11,21 @@ const projects: Array<
 > = [
   {
     browserName: "chromium",
-    user: { email: `testuser${process.env.TEST_RUN_ID}@openlogin.com` },
+    user: { emailSettings: `testuser${process.env.TEST_RUN_ID}@openlogin.com`,
+    emailLogin: `testuser${process.env.TEST_RUN_ID.split("").reverse().join("")}@openlogin.com`
```
Comment (tainguyentt) [inline openlogin/setup-2fa/index.config.ts:15]: pls format code

File: openlogin/setup-2fa/index.config.ts [hunk 2]
```diff
@@ -11,15 +11,21 @@ const projects: Array<
...
+   },
   },
   {
     browserName: "firefox",
-    user: { email: `testuser${process.env.TEST_RUN_ID}@openlogin.com` },
+    user: { emailSettings: `testuser${process.env.TEST_RUN_ID}@openlogin.com`,
+    emailLogin: `testuser${process.env.TEST_RUN_ID.split("").reverse().join("")}@openlogin.com`
```
Comment (tainguyentt) [inline openlogin/setup-2fa/index.config.ts:21]: pls format code

File: openlogin/setup-2fa/index.config.ts [hunk 3]
```diff
@@ -11,15 +11,21 @@ const projects: Array<
...
+   },
   },
   {
     browserName: "webkit",
-    user: { email: `testuser${process.env.TEST_RUN_ID}@openlogin.com` },
+    user: { emailSettings: `testuser${process.env.TEST_RUN_ID}@openlogin.com`,
+    emailLogin: `testuser${process.env.TEST_RUN_ID.split("").reverse().join("")}@openlogin.com`
```
Comment (tainguyentt) [inline openlogin/setup-2fa/index.config.ts:27]: pls format code

File: openlogin/setup-2fa/index.lib.ts [hunk 4]
```diff
@@ -3,10 +3,10 @@ import { env_map } from "../../utils/index";
 
 export interface TestArgs {
   openloginURL: string;
-  user: { email: string };
+  user: { emailSettings: string, emailLogin: string };
```
Comment (tainguyentt) [inline openlogin/setup-2fa/index.lib.ts:6]:
```suggestion
  user: { emailSettings: string; emailLogin: string };
```

File: openlogin/setup-2fa/index.test.ts [hunk 5]
```diff
@@ -35,49 +50,91 @@ async function setup2FA(page: Page, flow: string) {
   }
 }
 
+async function emailLogin(
+  browser: Browser, 
+  browserName: string, 
+  openloginURL: string, 
+  email: string,
+  page: Page){
```
Comment (tainguyentt) [inline openlogin/setup-2fa/index.test.ts:58]: pls format code

File: openlogin/setup-2fa/index.config.ts (hunk 1 above)
Comment (pandey-shubh) [reply openlogin/setup-2fa/index.config.ts]: done

File: openlogin/setup-2fa/index.config.ts (hunk 3 above)
Comment (pandey-shubh) [reply openlogin/setup-2fa/index.config.ts]: done

File: openlogin/setup-2fa/index.config.ts (hunk 2 above)
Comment (pandey-shubh) [reply openlogin/setup-2fa/index.config.ts]: done

**Review by tainguyentt — APPROVED** [review]
lgtm

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**